web/api/report_server.thrift

```diff
@@ -315,6 +321,7 @@ struct ReportData {
   15: i64             bugPathLength,   // Length of the bug path.
   16: optional ReportDetails details,  // Details of the report.
   17: optional string analyzerName,    // Analyzer name.
+  18: optional map<string, string> annotations, // Report annotations.
```

Comment:
I extended the documentation. However, I'm not sure if "which files to modify" should be included. I think an API documentation is for client applications about what information they can get and how they can get it.

Reply:
Since this is a string-string map, it is critical for the server to validate whether the message is proper. I think at the very least, lets bring attention to the importance of validation in `__validate_and_add_report_annotations`.